science/shtools/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                shtools
+revision            0
+
+categories          science math
+platforms           darwin
+license             BSD-3-Clause
```

Comment:
In MacPorts we call this license "BSD". See https://trac.macports.org/wiki/PortfileRecipes#license. We don't use SPDX identifiers yet. See https://trac.macports.org/ticket/61237.